runtime/binding-kafka/src/test/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/CacheFetchIT.java

```diff
@@ -453,6 +453,18 @@ public void shouldReceiveMessagesWithNoFilter() throws Exception
         k3po.finish();
     }
 
+    @Test
+    @Configuration("cache.yaml")
+    @Specification({
+        "${app}/cache.filter.sync/client",
+        "${app}/cache.filter.sync/server"})
+    @ScriptProperty("serverAddress \"zilla://streams/app1\"")
+    public void shouldFetchLatestMessage() throws Exception
+    {
+        partition.append(0L);
+        k3po.finish();
+    }
+
```

Comment:
We shouldn't need any cache-specific scripts, let's try to use a generic `filter.sync` script instead.